RickJim.kt

```diff
@@ -0,0 +1,38 @@
+//Con la doble barra creas un comentario de 1 sola línea
+/*
+Con estos símbolos se crea el comentario de varias líneas
+En esta segunda línea agrego la web de Kotlin que es https://kotlinlang.org/
+*/
+
+//constantes y variables en Kotlin
+val constante = "Soy una constante porque llevo val delante"
```

Comment:
i think it's a good practice to start coding in english :)

Reply:
Yeah it's a good practice! But this is for a spanish programming community in which most of us are learning, so it's for this the code is in Spanish. Anyway there's a problem, so this PR is gonna be deleted and I'll try to send again, but in spanish again